app/controllers/pieces_controller.rb

```diff
@@ -7,17 +10,34 @@ def show
 
   def update
     @selected_piece = ChessPiece.find(params[:id])
+    @game = @selected_piece.game
 
-    if @selected_piece && @selected_piece.valid_move?(Coordinates.new(move_to_x_parameter, move_to_y_parameter))
+    if moving_validly?
       @selected_piece.move_to!(Coordinates.new(move_to_x_parameter, move_to_y_parameter))
-      redirect_to piece_path(@selected_piece)
+      redirect_to game_path(@game)
     else
-      render 'Invalid', status: :unprocessable_entity
+      render text: 'Forbidden', status: :unauthorized
```

Comment:
Wouldn't it be better to re-render the show action for invalid moves instead of showing an error? Not a big deal though as this is just temporary.